src/test/java/Homework18.java

```diff
@@ -11,15 +11,16 @@
 public class Homework18 extends BaseTest {
     @Test
     public void playSongTest() throws InterruptedException {
-        WebDriver driver = new ChromeDriver();
         logIn("regniermandy@gmail.com", "te$t$tudent");
         playSong();
-    Assert.assertTrue(isSongPlaying());
+        Assert.assertTrue(isSongPlaying());
+
 }
 
     @Test
     public boolean isSongPlaying() {
```

Comment:
public boolean isSongPlaying() is not test, because one test - one assert

For example isSongPlaying() is simple method

![image](https://user-images.githubusercontent.com/104761482/222946266-b4392e5c-2bcf-4e17-807f-45c753824ab6.png)